aospy/test/test_data_loader.py

```diff
@@ -442,6 +443,26 @@ def test_load_variable(self):
         expected = xr.open_dataset(filepath)['condensation_rain']
         np.testing.assert_array_equal(result.values, expected.values)
 
+    def test_load_variable_non_0001_refdate(self):
+        def preprocess(ds, **kwargs):
+            ds['time'] = ds['time'] - 1095.
```

Comment:
Can you add a comment explaining the significance/reasoning for the 1095 value?  And/or define it as a constant.